### PR TITLE
Page names normalized

### PR DESCRIPTION
**Pull request by pavel-temtsunik**

### CODE REVIEW

File: features/functionality/about_blank.feature [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @bvt
-Feature: Blank page
+Feature: blank page
```
Comment (romikoops) [inline features/functionality/about_blank.feature]: Em, feature name should start from Capital letter.

File: features/functionality/articles/adding.feature [hunk 2]
```diff
@@ -9,29 +9,29 @@ Feature: Article Adding
       | user_name | UNIQ_USER[:full_name] |
       | email     | UNIQ_USER[:email]     |
       | password  | UNIQ_USER[:password]  |
-    And opened Articles list page
+    And opened articles list page
 
   Scenario: user can add article with correct credentials
-    When I click New article button on Articles list page
-    And I fill New article form on New article page with data:
+    When I click New article button on articles list page
```
Comment (romikoops) [inline features/functionality/articles/adding.feature]: please change form New as well

File: features/functionality/articles/adding.feature [hunk 3]
```diff
@@ -9,29 +9,29 @@ Feature: Article Adding
...
+    And I fill New article form on new article page with data:
       | title     | UNIQ_ARTICLE[:title]  |
       | text      | UNIQ_ARTICLE[:text]   |
-    And I submit New article form on New article page
-    Then I should see article parameters on Article page with data:
+    And I submit New article form on new article page
```
Comment (romikoops) [inline features/functionality/articles/adding.feature]: new

File: features/functionality/articles/adding.feature [hunk 4]
```diff
@@ -9,29 +9,29 @@ Feature: Article Adding
...
   Scenario: user can not add article with blank field
-    When I click New article button on Articles list page
-    And  I fill New article form on New article page with blank data:
+    When I click New article button on articles list page
+    And  I fill New article form on new article page with blank data:
       | title     |                       |
       | text      | UNIQ_ARTICLE[:text]   |
-    And I submit New article form on New article page
-    Then I should see following text on New article page:
+    And I submit New article form on new article page
```
Comment (romikoops) [inline features/functionality/articles/adding.feature]: new

File: features/functionality/articles/adding.feature [hunk 5]
```diff
@@ -39,12 +39,12 @@ Feature: Article Adding
       """
 
   Scenario: user can not add article with title is too short
-    When I click New article button on Articles list page
-    And I fill New article form on New article page with data:
+    When I click New article button on articles list page
+    And I fill New article form on new article page with data:
```
Comment (romikoops) [inline features/functionality/articles/adding.feature]: new

File: features/functionality/articles/comments/adding.feature [hunk 6]
```diff
@@ -15,22 +15,22 @@ Feature: Article Comment Adding
...
       | commenter | UNIQ_USER[:email]     |
       | comment   | UNIQ_COMMENT[:text]   |
 
   Scenario: user can not add comment with blank comment body
-    When I fill New Comment form on Article page with blank data:
+    When I fill New Comment form on article page with blank data:
       | body      |                       |
-    And I submit New Comment form on Article page
-    Then I should see following text on Article page:
+    And I submit New Comment form on article page
```
Comment (romikoops) [inline features/functionality/articles/comments/adding.feature]: new

File: features/functionality/articles/comments/list.feature [hunk 7]
```diff
@@ -2,7 +2,7 @@
 Feature: Article Comments List
   As user
   I want to view my article comments 
-  So I can view my article comments on Article comments list on Article page
+  So I can view my article comments on Article comments list on article page
```
Comment (romikoops) [inline features/functionality/articles/comments/list.feature]: article

File: features/functionality/articles/viewing.feature [hunk 8]
```diff
@@ -23,8 +23,8 @@ Feature: Article Viewing
     And I should see comment parameters with data:
       | commenter | UNIQ_USER[:email]     |
       | comment   | UNIQ_COMMENT[:text]   |
-    And I should see Add comment form on Article page
-    And I should see Body field on Article page
-    And I should see buttons: Edit article, Destroy comment, Create comment on Article page
-    When I click Back to articles link on Article page
-    Then I should be redirected to Articles list page
\ No newline at end of file
+    And I should see Add comment form on article page
```
Comment (romikoops) [inline features/functionality/articles/viewing.feature]: add

File: features/functionality/articles/viewing.feature [hunk 9]
```diff
@@ -23,8 +23,8 @@ Feature: Article Viewing
...
+    And I should see Body field on article page
```
Comment (romikoops) [inline features/functionality/articles/viewing.feature]: body

File: features/functionality/articles/viewing.feature [hunk 10]
```diff
@@ -23,8 +23,8 @@ Feature: Article Viewing
...
+    And I should see buttons: Edit article, Destroy comment, Create comment on article page
```
Comment (romikoops) [inline features/functionality/articles/viewing.feature]: edit, destroy

File: features/functionality/articles/viewing.feature [hunk 11]
```diff
@@ -23,8 +23,8 @@ Feature: Article Viewing
...
+    When I click Back to articles link on article page
```
Comment (romikoops) [inline features/functionality/articles/viewing.feature]: back

File: features/functionality/auth/login.feature [hunk 12]
```diff
@@ -4,10 +4,10 @@ Feature: Log In
   So I can login the system
 
   @bvt
-  Scenario: user can open Login page via menu
-    Given Home page of web application
-    When I click Login menu item on Home page
-    Then Login page should be displayed
+  Scenario: user can open login page via menu
+    Given home page of web application
+    When I click Login menu item on home page
```
Comment (romikoops) [inline features/functionality/auth/login.feature]: login

File: features/functionality/auth/login.feature [hunk 13]
```diff
@@ -29,22 +29,22 @@ Feature: Log In
...
+    When I fill form on login page with data:
       | email     | UNIQ_USER[:email]     |
       | password  | UNIQ_USER[:password]  |
     And I check 'Remember me' checkbox
-    And I submit form on Login page
+    And I submit form on login page
     Then I should be logged in the system
-    And I should be redirected to Home page
+    And I should be redirected to home page
     When I click Logout menu item
```
Comment (romikoops) [inline features/functionality/auth/login.feature]: Logout

File: features/functionality/auth/login.feature [hunk 14]
```diff
@@ -29,22 +29,22 @@ Feature: Log In
...
     And I click Login menu item
```
Comment (romikoops) [inline features/functionality/auth/login.feature]: Login

File: features/functionality/auth/login.feature [hunk 15]
```diff
@@ -151,18 +151,18 @@ Feature: Log In
       | user_name | UNIQ_USER[:full_name] |
       | email     | UNIQ_USER[:email]     |
       | password  | UNIQ_USER[:password]  |
-    And Edit account page of web application
-    When I click 'Cancel account' item on Edit account page
+    And edit account page of web application
+    When I click 'Cancel account' item on edit account page
```
Comment (romikoops) [inline features/functionality/auth/login.feature]: Please replace with your new step

File: features/functionality/auth/signup.feature [hunk 16]
```diff
@@ -5,109 +5,109 @@ Feature: Sign Up
   So I can login with a new account
   
   @bvt
-  Scenario: user can open Sign up page via menu
-    Given Home page of web application
-    When I click Sign up menu item on Home page
-    Then I should be redirected to Sign up page
+  Scenario: user can open sign up page via menu
+    Given home page of web application
+    When I click Sign up menu item on home page
```
Comment (romikoops) [inline features/functionality/auth/signup.feature]: sign

File: features/functionality/auth/signup.feature [hunk 17]
```diff
@@ -5,109 +5,109 @@ Feature: Sign Up
...
-    Given Sign up page of web application
-    When I fill form on Sign Up page with data:
+    Given sign up page of web application
+    When I fill form on sign up page with data:
       | user_name             |         |
       | email                 |         |
       | password              |         |
       | password_confirmation |         |
-    And I submit Sign up form on Sign up page
+    And I submit Sign up form on sign up page
```
Comment (romikoops) [inline features/functionality/auth/signup.feature]: sign

File: features/functionality/auth/signup.feature [hunk 18]
```diff
@@ -5,109 +5,109 @@ Feature: Sign Up
...
     1 error prohibited this user from being saved: Password can't be blank
     """
-    When I fill form on Sign Up page with data:
+    When I fill form on sign up page with data:
       | user_name             |                      |
       | email                 |                      |
       | password              | UNIQ_USER[:password] |
       | password_confirmation | UNIQ_USER[:password] |
-    And I submit Sign up form on Sign up page
+    And I submit Sign up form on sign up page
```
Comment (romikoops) [inline features/functionality/auth/signup.feature]: sign

File: features/functionality/auth/signup.feature [hunk 19]
```diff
@@ -5,109 +5,109 @@ Feature: Sign Up
...
      1 error prohibited this user from being saved: Password is too short (minimum is 8 characters)
      """
-    When I fill form on Sign Up page with data:
+    When I fill form on sign up page with data:
       | user_name             |                      |
       | email                 | UNIQ_USER[:email]    |
       | password              | 1234567890           |
       | password_confirmation | 1234567890123        |
-    And I submit Sign up form on Sign up page
+    And I submit Sign up form on sign up page
```
Comment (romikoops) [inline features/functionality/auth/signup.feature]: sign

File: features/permissions/user.feature [hunk 20]
```diff
@@ -42,25 +42,25 @@ Feature: User Permissions
       | commenter | UNIQ_USER2[:email]      |
       | comment   | UNIQ_COMMENT2[:text]    |
 
-  Scenario: user can open Users list page
+  Scenario: user can open users list page
     When I click on Users menu item
```
Comment (romikoops) [inline features/permissions/user.feature]: Users

File: features/permissions/visitor.feature [hunk 21]
```diff
@@ -4,23 +4,23 @@ Feature: Visitor Permissions
...
 
-  Scenario: visitor can view Home page
+  Scenario: visitor can view home page
     When I click Home menu item 
-    Then I should be redirected to Home page
+    Then I should be redirected to home page
 
-  Scenario: visitor can view Articles list page
+  Scenario: visitor can view articles list page
     When I click Articles menu item
```
Comment (romikoops) [inline features/permissions/visitor.feature]: articles

File: features/step_definitions/sign_up_steps.rb [hunk 22]
```diff
@@ -1,6 +1,6 @@
 ####################################
 #              ACTIONS             #
 ####################################
-When /^I submit Sign up form on Sign up page$/ do
+When /^I submit Sign up form on sign up page$/ do
```
Comment (romikoops) [inline features/step_definitions/sign_up_steps.rb]: sign

File: features/functionality/about_blank.feature (hunk 1 above)
Comment (pavel-temtsunik) [inline features/functionality/about_blank.feature]: +

File: features/functionality/articles/adding.feature (hunk 2 above)
Comment (pavel-temtsunik) [inline features/functionality/articles/adding.feature]: +

File: features/functionality/articles/comments/adding.feature (hunk 6 above)
Comment (pavel-temtsunik) [inline features/functionality/articles/comments/adding.feature]: +

File: features/functionality/articles/comments/list.feature (hunk 7 above)
Comment (pavel-temtsunik) [inline features/functionality/articles/comments/list.feature]: +

File: features/functionality/articles/viewing.feature (hunk 8 above)
Comment (pavel-temtsunik) [inline features/functionality/articles/viewing.feature]: +

File: features/functionality/articles/viewing.feature (hunk 9 above)
Comment (pavel-temtsunik) [inline features/functionality/articles/viewing.feature]: +

File: features/functionality/articles/viewing.feature (hunk 10 above)
Comment (pavel-temtsunik) [inline features/functionality/articles/viewing.feature]: +

File: features/functionality/articles/viewing.feature (hunk 11 above)
Comment (pavel-temtsunik) [inline features/functionality/articles/viewing.feature]: +

File: features/functionality/auth/login.feature (hunk 12 above)
Comment (pavel-temtsunik) [inline features/functionality/auth/login.feature]: +

File: features/functionality/auth/login.feature (hunk 13 above)
Comment (pavel-temtsunik) [inline features/functionality/auth/login.feature]: +

File: features/functionality/auth/login.feature (hunk 14 above)
Comment (pavel-temtsunik) [inline features/functionality/auth/login.feature]: +

File: features/functionality/auth/login.feature (hunk 15 above)
Comment (pavel-temtsunik) [inline features/functionality/auth/login.feature]: not only one step. I have adeed necessary steps as: 
 When I fill form on login page with data:
         | email     | UNIQ_USER[:email]     |
         | password  | UNIQ_USER[:password]  |
    And I submit form on login page
    And I should be logged in the system
    And I should be redirected to home page
    And I open edit account page
    And I cancel account on edit account page
    And I confirm account cancelling on edit account page

In another case -user will be not logged in, and will not be able to cancel his account

File: features/functionality/auth/signup.feature (hunk 16 above)
Comment (pavel-temtsunik) [inline features/functionality/auth/signup.feature]: +

File: features/functionality/auth/signup.feature (hunk 17 above)
Comment (pavel-temtsunik) [inline features/functionality/auth/signup.feature]: +

File: features/functionality/auth/signup.feature (hunk 18 above)
Comment (pavel-temtsunik) [inline features/functionality/auth/signup.feature]: +

File: features/functionality/articles/adding.feature (hunk 3 above)
Comment (pavel-temtsunik) [inline features/functionality/articles/adding.feature]: +

File: features/functionality/articles/adding.feature (hunk 4 above)
Comment (pavel-temtsunik) [inline features/functionality/articles/adding.feature]: +

File: features/functionality/articles/adding.feature (hunk 5 above)
Comment (pavel-temtsunik) [inline features/functionality/articles/adding.feature]: +

File: features/functionality/auth/signup.feature (hunk 19 above)
Comment (pavel-temtsunik) [inline features/functionality/auth/signup.feature]: +

**Pull request merged**